browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -732,6 +737,14 @@ void RewardsDOMHandler::RestorePublishers(const base::ListValue *args) {
   }
 }
 
+void RewardsDOMHandler::RestorePublisher(const base::ListValue *args) {
+  if (rewards_service_) {
+    std::string publisherKey;
+    args->GetString(0, &publisherKey);
+    rewards_service_->RestorePublisher(publisherKey);
```

Comment:
please add callback which will tell UI that publisher is restored and they need to fetch new lists